src/components/Vault/VaultTableBody.tsx

```diff
@@ -31,21 +34,34 @@ export default VaultTableBody
 
 const VaultsLoading: FC = () => {
   return (
-    <VaultTableRow
+    <TableRow
       className="flex h-44 items-center rounded-t-none"
-      aria-label="Loading vaults..."
+      aria-label="Loading compounders..."
     >
       <span className="col-span-full text-center" aria-hidden="true">
         <Spinner className="h-10 w-10" />
       </span>
-    </VaultTableRow>
+    </TableRow>
   )
 }
 
-const NoVaultsFound: FC = () => {
+const NoVaultsFound: FC<Pick<VaultProps, "type">> = ({ type }) => {
+  const chainId = useActiveChainId()
+  const availableChains = enabledNetworks.chains.filter((n) => n.id === chainId)
+  const network = availableChains?.[0].name
   return (
-    <VaultTableRow className="flex h-44 items-center rounded-t-none">
-      <h2 className="col-span-full text-center text-2xl">No vaults found...</h2>
-    </VaultTableRow>
+    <TableRow className="flex h-44 items-center rounded-t-none">
+      <div className="col-span-full grid grid-rows-[1fr,1fr]">
```

Comment:
You don't really need to use `grid` for this and it actually has a weird effect on mobile. I'd just remove the grid classes and then add `mb-3` to the `h2`. You'll want to do this on the holdings message too so they are visually consistent.